spec/lib/tasks/import_spec.rb

```diff
@@ -24,22 +24,34 @@
     let(:whitehall_export) { build(:whitehall_export_document) }
 
     before do
-      allow($stdout).to receive(:puts)
       Rake::Task["import:whitehall_document"].reenable
-      stub_request(:get, "#{whitehall_host}/government/admin/export/document/123")
-        .to_return(status: 200, body: whitehall_export.to_json)
     end
 
     it "imports the export and syncs with publishing-api" do
-      import = build(:whitehall_migration_document_import, state: :completed)
+      import = build(:whitehall_migration_document_import, state: "completed")
+
+      allow(WhitehallMigration::DocumentImport).to receive(:create!).and_return(import)
+      allow(WhitehallImporter).to receive(:import_and_sync).and_return(import)
```

Comment:
We shouldn't care that this returns anything anymore.

Reply:
I've removed the return value since it's not needed